src/engine/bank03.asm

```diff
@@ -785,7 +785,7 @@ TextIDs_d18f:
 	tx Text09cd ; OWMAP_GR_PSYCHIC_STRONGHOLD
 	tx Text09ce ; OWMAP_COLORLESS_ALTAR
 	tx Text09cf ; OWMAP_GR_CASTLE
-	tx Text0069
+	tx Drew7CardsText ; $0069, weird
```

Comment:
Weird indeed. I wonder if this pointer table shouldn't include this, and these bytes are the start of something else..